src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/__tests__/SavedSearchAlertEditForm.jest.tsx

```diff
@@ -120,6 +129,42 @@ describe("SavedSearchAlertEditForm", () => {
     await waitFor(() => expect(mockOnCompleted).toBeCalled())
   })
 
+  it("should track editedSavedSearch event when alert info is successfully updated", async () => {
+    renderWithRelay({
+      Artist: () => artistMocked,
+      FilterArtworksConnection: () => filterArtworksConnectionMocked,
+      SearchCriteria: () => savedSearchAlertMocked,
+    })
+
+    fireEvent.change(screen.getByDisplayValue("Alert #1"), {
+      target: { value: "Updated Name" },
+    })
+
+    const saveAlertButton = screen.getByRole("button", {
+      name: "Save Alert",
+    })
+
+    fireEvent.click(saveAlertButton)
+
+    await waitFor(() => expect(mockOnCompleted).toBeCalled())
+
+    expect(trackEvent).toBeCalledWith({
+      action: "editedSavedSearch",
+      context_screen_owner_type: "savedSearch",
+      context_screen_owner_id: "alert-id",
+      current: JSON.stringify({
+        name: "Alert #1",
+        email: true,
+        push: false,
+      }),
+      changed: JSON.stringify({
+        name: "Updated Name",
+        push: false,
+        email: true,
+      }),
+    })
```

Comment:
**suggestion(non-blocking):** what do you think of using `toMatchInlineSnapshot` here as we do in other analytics assertions? (like [this one](https://github.com/artsy/force/blob/c28fc18c856513a7d201886513c80e213ff916da/src/v2/Components/Authentication/__tests__/FormSwitcher.jest.tsx#L138-L152))

Reply:
Done